cmd/sanitize-prow-jobs/main.go

```diff
@@ -75,6 +93,10 @@ func determinizeJobs(prowJobConfigDir string, config *dispatcher.Config) error {
 
 			defaultJobConfig(jobConfig, path, config)
 
+			if dtEnforcer.Enabled {
+				dtEnforcer.IngestAll(jobConfig)
```

Comment:
Also, I would have expected the `IngestAll` call to be made part of the be executed in the `GetEnforcer` call, its not obvious from the public api that this must be called first and IMHO it doesn't really make sense to force consumers of the Enforcer to know this